packages/sdk/client-services/src/packlets/services/service-host.ts

```diff
@@ -243,7 +244,36 @@ export class ClientServicesHost {
       SystemService: this._systemService,
 
       IdentityService: new IdentityServiceImpl(
-        (params) => this._serviceContext.createIdentity(params),
+        async (params) => {
+          const identity = await this._serviceContext.createIdentity(params);
+
+          // Setup default space.
+          await this._serviceContext.initialized.wait();
+          const space = await this._serviceContext.dataSpaceManager!.createSpace();
+          const obj: TypedObject = new Properties();
```

Comment:
I remember there was a place where Properties object was auto-created for new spaces. Do we need to remove that code since its here now?

Reply:
That appears to be here https://github.com/dxos/dxos/blob/main/packages/sdk/client/src/echo/space-list.ts#L171 which is outside the code path for the default space so I think we're good.

Reply:
👍